packages/next/src/server/app-render/app-render.tsx

```diff
@@ -2055,7 +2401,10 @@ async function prerenderToStream(
           } else if (prospectiveRenderFlightSignal.aborted) {
             reactServerIsDynamic = true
             return PRERENDER_COMPLETE
-          } else if (process.env.NEXT_DEBUG_BUILD) {
+          } else if (
+            process.env.NEXT_DEBUG_BUILD ||
```

Comment:
I made up this verbose logging thing. I think we should have it and it applies wider than just builds. Leaving the old env around to activate this as well for now